source/_header.html.erb

```diff
@@ -0,0 +1,8 @@
+  <nav>
+    <ul class="main-nav">
+      <li class="main-nav__name"><span class="main-nav__name--bold">Jeff</span>Stolz</li>
+        <li class="main-nav__link"><%= link_to 'Work', '/work.html' %></li>
```

Comment:
The `main-nav__name--bold` seems very specific. If this is a class that makes type bold, I'd suggest considering creating a utility class to handle that. Then you could reuse it throughout the site. Something like `u-text--is-bold` or `u-text--bold` something in that vein. The `u` is a prefix, meaning utility. I often use that for clarity.

Reply:
So some context on this one - The first list item is my left anchored "logo." Does it make sense to include this within the ul of the other nav items, or separate it out?

![screen shot 2018-10-12 at 10 29 44 am](https://user-images.githubusercontent.com/2924479/46875592-101b0080-ce0a-11e8-8173-0e8f74b3e447.png)

Reply:
I understand how it's currently structured can be confusing. Is there also a more DRY way to apply the "main-nav__link" class rather than repeating it for each li item?

Reply:
What does your heart tell you to do? I'd look at some other Header / Navs and see how they're structured. For me, I chose a few people who seemed like they were smart and looked and copied and learned from their work. Teach a man to fish ... and all that :)

There ARE times when you aren't able to add classes, but if you are able you should. The moment you being targeting nested element selectors, you're creating problems for yourself and/or others. Keeping CSS as flat as possible should be the goal.

Reply:
> The `main-nav__name--bold` seems very specific. If this is a class that makes type bold, I'd suggest considering creating a utility class to handle that. Then you could reuse it throughout the site. Something like `u-text--is-bold` or `u-text--bold` something in that vein. The `u` is a prefix, meaning utility. I often use that for clarity.

This is actually a unique element, so I'll rename it to prevent this confusion